features/unattached_commands.feature

```diff
@@ -84,6 +84,13 @@ Feature: Command behaviour when unattached
         """
         apt-esm-json-hook
         """
+        When I create the file `/etc/apt/sources.list.d/empty-release-origin.list` with the following
+        """
+        deb [ allow-insecure=yes ] https://packages.irods.org/apt xenial main
+        """
+        Then I verify that running `apt-get update` `with sudo` exits `0`
+        When I run `python3 /usr/lib/ubuntu-advantage/ua_update_messaging.py` with sudo
+        Then I verify that running `grep Traceback /var/log/ubuntu-advantage.log` `with sudo` exits `1`
```

Comment:
I get the idea of the test, but this check can be a problematic since unrelated `Traceback`  in the log will make the test fail. I think we can actually call the hook binary in the test and see that it exits `0`

Reply:
+1 bumped it to run /usr/lib/ubuntu-advantage/apt-esm-hook process-templates directly